server/scripts/randomize-participants.ts

```diff
@@ -31,6 +31,7 @@ const generateParticipants = async (amount: number) => {
         preferredLocation: DEFAULT_PREF_LOCATION,
         interested: 'yes',
         crcClear: 'yes',
+        program: Math.floor(Math.random() * 2) === 0 && 'MHAW', //NOSONAR
```

Comment:
Should we include 'HCA'? We can use `_.sample`.

Reply:
oh right yes, I thought it was HCA by default, but I was wrong in how this worked, will add